ddtrace/contrib/flask/__init__.py

```diff
@@ -108,6 +108,6 @@ def index():
 
         patch = _patch.patch
         unpatch = _patch.unpatch
-        get_version = _patch.get_version
+        get_version = _patch._get_version
```

Comment:
We shouldn't change this. The get_version we expose here should raise a deprecation warning.